src/main/java/hudson/plugins/plot/PlotDescriptor.java

```diff
@@ -1,73 +0,0 @@
-package hudson.plugins.plot;
-
-import hudson.Extension;
-import hudson.FilePath;
-import hudson.matrix.MatrixProject;
-import hudson.model.AbstractProject;
-import hudson.tasks.BuildStepDescriptor;
-import hudson.tasks.Publisher;
-import hudson.util.FormValidation;
-
-import java.io.IOException;
-
-import net.sf.json.JSONObject;
-
-import org.kohsuke.stapler.AncestorInPath;
-import org.kohsuke.stapler.QueryParameter;
-import org.kohsuke.stapler.StaplerRequest;
-
-/**
- * The Descriptor for the plot configuration Extension
- *
- * @author Nigel Daley
- * @author Thomas Fox
- *
- */
-@Extension
-public class PlotDescriptor extends BuildStepDescriptor<Publisher> {
```

Comment:
This will be a problem if any plugin depends on the APIs from the plot plugin - do any of them?

Reply:
This kind of change makes me a little nervous about potential impacts when loading configurations created from older versions of the plugin.  I can't say 100% if it will be a problem or will be okay.